yarn.lock

```diff
@@ -3506,6 +3506,10 @@ grid-index@^1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/grid-index/-/grid-index-1.0.0.tgz#ad2c5d54ce5b35437faff1d70a9aeb3d1d261110"
 
+h3-js@^3.1.0:
```

Comment:
Please update your `~/.npmrc` to use `registry = https://registry.yarnpkg.com/` before running yarn. This is why the build failed